src/components/Hero.tsx

```diff
@@ -1,9 +1,11 @@
 const Hero = () => (
-  <section aria-labelledby="hero-heading" className="flex flex-col py-24 items-center mx-2.5">
+  <section
+    aria-labelledby="hero-heading"
```

Comment:
refactor (blocking): assistive technologies know how to pick up the header within the section, so no need for aria.

```suggestion
  <section
```

Reply:
thanks a lot for the information 😊